config/crd/bases/test.openstack.org_horizontests.yaml

```
@@ -132,10 +203,9 @@ spec:
             - dashboardUrl
             - horizonRepoBranch
             - repoUrl
-            - storageClass
             type: object
           status:
-            description: HorizonTestStatus defines the observed state of HorizonTest
+            description: CommonTestStatus defines the observed state of the controller
```

I know that this is a nit picking but since this is about cleaning up the CRs I'm trying to pick all the problems. `CommonTestStatus` is not a name that is facing the users of the test-operator but `status`. Try using `oc explain horizontest.status`.

```suggestion
            description: Status defines the observed state of the controller
```